frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/dialogs/SlideInDialog.kt

```diff
@@ -105,4 +123,14 @@ class SlideInDialog
       startTrackingValidation()
     }
   }
+
+  private fun isDescendant(
```

Comment:
Given that in both invocations of `isDescendant`, second param is just `it`, consider currying this method (https://en.proft.me/2018/08/12/currying-function-kotlin/)